pkg/services/app/remove_test.go

```diff
@@ -0,0 +1,134 @@
+package app
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	wego "github.com/weaveworks/weave-gitops/api/v1alpha1"
+)
+
+var workloadYaml1 = []byte(`apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-nginx
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: my-nginx
+  labels:
+    name: nginx
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      name: nginx
+  template:
+    metadata:
+      namespace: my-nginx
+      labels:
+        name: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        ports:
+        - containerPort: 80
+`)
+
+var workloadYaml2 = []byte(`apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-nginx2
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx2
+  namespace: my-nginx2
+  labels:
+    name: nginx2
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      name: nginx2
+  template:
+    metadata:
+      namespace: my-nginx2
+      labels:
+        name: nginx2
+    spec:
+      containers:
+      - name: nginx2
+        image: nginx
+        ports:
+        - containerPort: 80
+`)
+
+var application wego.Application
+
+func populateAppRepo() (string, error) {
+	dir, err := ioutil.TempDir("", "an-app-dir")
+	if err != nil {
+		return "", err
+	}
+
+	workloadPath1 := filepath.Join(dir, "kustomize", "one", "path", "to", "files")
+	workloadPath2 := filepath.Join(dir, "kustomize", "another", "path", "to", "more", "files")
+	if err := os.MkdirAll(workloadPath1, 0777); err != nil {
+		return "", err
+	}
+	if err := os.MkdirAll(workloadPath2, 0777); err != nil {
+		return "", err
+	}
+
+	if err := ioutil.WriteFile(filepath.Join(workloadPath1, "nginx.yaml"), []byte(workloadYaml1), 0644); err != nil {
```

Comment:
it's a minor thing, but we don't need to specify the whole file definition to verify if the Remove is working. you could just pass the string here `[]byte("file1")` and the same later and check it in the test.

I think that makes clear what is being actually tested and clean up the code since we don't actually need the actual definition for anything else.

Reply:
That makes sense; you're right, the file contents are irrelevant.